docs/rfcs/0002_templates_mvp.md

```diff
@@ -0,0 +1,339 @@
+| All fields in DB table schema |
+| template_versions             | RelatedManager | List of template versions this template has |
+
+### TemplateVersion
+
+A `TemplateVersion` is a container for the tabular submission file
+(`submission_template_file`) and associated submission rules
+(`fields_schema`). See [Definition](#📝-definition) for more details.
+
+#### Database Table Schema
+
+| Name          | Type        | Description                                                                                                                                                                                        |
+| ------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id            | varchar     | A human friendly identifier (Kids First style) for the submission template                                                                                                                         |
+| description   | text        | A short blurb describing this version of the template                                                                                                                                              |
+| fields_schema | JSON        | The template submission rules. A JSON blob with a standard structure which will encode the template field objects having things like: label, key, description, type, values_accepted, instructions |
+| created_at    | datetime    | Timestamp of creation                                                                                                                                                                              |
+| modified_at   | varchar     | Timestamp of update                                                                                                                                                                                |
+| creator       | foreign key | Foreign key to User that created the submission template                                                                                                                                           |
+
+#### Django Model Schema
+
+| Name                          | Type           | Description                                                                                            |
+| ----------------------------- | -------------- | ------------------------------------------------------------------------------------------------------ |
+| All fields in DB table schema |
+| released                      | BooleanField   | A computed field (not stored in db) denoting whether the template is being used in one or more studies |
+| submission_template_file      | FileField      | Reference to the actual template file in a storage system                                              |
```

Comment:
Yea, I was thinking the submission template would be computed from the fields schema. It might look something like this: 
<img width="1438" alt="image" src="https://user-images.githubusercontent.com/10283712/119037374-e510ca00-b97f-11eb-95ff-0b03639e9440.png">

Reply:
In this case, we shouldn't need to store anything in the database since we can generate it on demand.

Reply:
Ok, wasn't sure. I'll update the doc